perun-cli/Perun/auth/OidcAuth.pm

```diff
@@ -268,22 +289,78 @@ sub loadAccessToken
 
 sub removeAccessToken
 {
+	if (defined $_[0]) {
+		$PERUN_OIDC = "perun_oidc" . "_" . $_[0];
+	}
+
 	`$PYTHON -c "import keyring; keyring.delete_password('$PERUN_OIDC', 'access_token')"`;
 	`$PYTHON -c "import keyring; keyring.delete_password('$PERUN_OIDC', 'access_token_validity')"`;
 }
 
+sub removeAllAccessTokens
+{
+	my %config = % { loadAllConfigurations() };
+	foreach my $key (keys %config) {
+		if (getAccessToken($key)) {
+			removeAccessToken($key);
+			print("Access token from configuration $key successfully removed from keyring.\n");
```

Comment:
I think we can omit printing in this file because we also print it in separate scripts where we also consider whether batch mode is on (if it is on, nothing should be printed). What do you think @Johaney-s @xflord ?

Reply:
I’m not too familiar with how batch mode but I will remove the prints in OidcAuth. Question is, do we want to move the prints into the scripts or is the amount of information provided by them enough?